mirrord/operator/src/setup.rs

```diff
@@ -105,6 +106,7 @@ pub struct Operator {
     namespace: OperatorNamespace,
     role: OperatorRole,
     role_binding: OperatorRoleBinding,
+    kafka_secret_role: Option<OperatorSecretRole>,
```

Comment:
```suggestion
    role: Option<OperatorSecretRole>,
```
plus rename exisiting `role` to `cluster_role`